### DISCUSSION TITLE
Deploy with Screen

### DISCUSSION THREAD

**Discussion by Fotguedes** [anasty17/mirror-leech-telegram-bot#743] [Q&A]
**Is your feature request related to a problem? Please describe.**
I have a seedbox where i'm not allowed to use docker and sudo commands

**Describe the solution you'd like**
Most of the softwares I need to keep running in the seedbox they allow to use the command "screen" and keep it running in the background

**Describe alternatives you've considered**
I would like to know if it's possible to deploy the software in the seedbox (whatbox) using CLI commands and the screen command

**Comment by anasty17** [comment]
If u can't use sudo how u will install packages ! I don't have any idea

**Reply by Fotguedes** [reply]
I already managed to install all the rerequisites, since they already provide most of the apps, the only problem I had was to deploy and also to create the token.picle, since when I try to authorize in Google it redirects me to a localhost

**Reply by anasty17** [reply]
What do u have installed? I'm not talking about requirements.txt

**Reply by Fotguedes** [reply]
I'm usually able to install most of the packages with python, I don't know what else I would need to install to try using it through CLI, but for now I have everything except Docker installed.

When running the docker what packages does it ask to install? I could try installing then here to see if i can do it.

**Reply by anasty17** [reply]
If u can't install docker then how u will install the packages?

**Reply by anasty17** [reply]
Packages like p7zip-full aria2 qbittorrent-nox

**Reply by anasty17** [reply]
And please explain what do u mean by deploy with screen ?!

**Reply by anasty17** [reply]
Which vps u r using explain..

**Reply by Fotguedes** [reply]
Sure, I will try to elaborate.

Some apps like tautulli that run through terminal i just run then with the `screen -dmS` command, so when I close the SSH connection it won't stop.

I was trying to know if it was possible to do this with the mirror-leech-bot, some apps like p7zip-full and qbittorrent are already at the machine, but the aria2 you gave as an example isn't and probably I will not be able to install.

I guess it would be possible to run with `screen` but most of the functions would no working... Sadly.

For me if the clone command worked it would be already perfect, but I think that with my current enviroment that is nothing I can do by now.

**Reply by Fotguedes** [reply]
It's not a VPS, that's the biggest problem haha it's http://whatbox.ca/

**Reply by anasty17** [reply]
I have no idea about this platform
